core/src/main/java/org/apache/carbondata/core/carbon/metadata/schema/table/CarbonTable.java

```diff
@@ -62,6 +62,11 @@
   private Map<String, List<CarbonDimension>> tableDimensionsMap;
 
   /**
+   * TableName, Dimensions and children dimensions list
+   */
+  private Map<String, List<CarbonDimension>> tableAllDimensionsMap;
+
+  /**
    * table measures list.
```

Comment:
what's the different between tableDimensionsMap and tableAllDimensionsMap, can merge them into one ?